code/obj/flock/structure/flock_structure_ghost.dm

```diff
@@ -13,7 +13,7 @@
 	return {"<span class='bold'>Construction Percentage:</span> [!src.goal == 0 ? round((src.currentmats/src.goal)*100) : 0]%
 	<br><span class='bold'>Construction Progress:</span> [currentmats] materials added, [goal] needed"}
 
-/obj/flock_structure/ghost/New(atom/location, obj/flock_structure/building = null, datum/flock/F = null, goal = 0)
+/obj/flock_structure/ghost/New(atom/location, datum/flock/F, obj/flock_structure/building = null, goal = 0)
```

Comment:
```suggestion
/obj/flock_structure/ghost/New(atom/location, datum/flock/F, atom/converted_from = null, obj/flock_structure/building = null, goal = 0)
```
To stay consistent with the parent, obviously need to modify the code that calls this too.